simulator/config.yaml

```diff
@@ -38,12 +38,24 @@ kubeSchedulerConfigPath: ""
 # This variable indicates whether the simulator will
 # import resources from a user cluster specified by kubeConfig.
 # Note that it only imports the resources once when the simulator is started.
-# You cannot make both externalImportEnabled and resourceSyncEnabled true because those features would be conflicted.
+# You cannot make two or more of externalImportEnabled, resourceSyncEnabled and replayEnabled true
+# because those features would be conflicted.
 # This is still a beta feature.
 externalImportEnabled: false
 
 # This variable indicates whether the simulator will
 # keep syncing resources from an user cluster's or not.
-# You cannot make both externalImportEnabled and resourceSyncEnabled true because those features would be conflicted.
+# You cannot make two or more of externalImportEnabled, resourceSyncEnabled and replayEnabled true
+# because those features would be conflicted.
 # Note, this is still a beta feature.
 resourceSyncEnabled: false
+
+# This variable indicates whether the simulator will
+# replay events recorded in the file specified by recordFilePath.
+# You cannot make two or more of externalImportEnabled, resourceSyncEnabled and replayEnabled true
+# because those features would be conflicted.
+# Note, this is still a beta feature.
+replayEnabled: false
+
+# The path to a file where the record files are stored.
+recordFilePath: "/record.jsonl"
```

Comment:
```suggestion
recordFilePath: "/record.json"
```

Reply:
The records are serialized into JSON Lines. So the extension may be `jsonl`.

https://jsonlines.org/